scripts/populate.js

```diff
@@ -3,7 +3,17 @@ var Chatty = (function(originalChatty){
 	var outputString = "";
 	var messagesArray = [];
 
-	
+	Chatty.getMessage = function () {
+		return messagesArray;
+	}
+
+	Chatty.newMessage = function () {
+		messagesArray.push(inputEl.value);
+		for (var i = 0; i < messagesArray.length; i++) {
+			outputString = messagesArray[i];
+		};
+			messageContainer.innerHTML += "<div id='message" + [i] + "' class='newMessages'>" + "User: " + outputString + "<button class='deleteButton'>Delete</button>" + "</div>";
+	}
```

Comment:
These comments went in in a weird order, so this is kind of a mix of your lines 13 and 15. I also used outputString as a variable, but I set it's value this way:
```js
outputString = `<div id="${messagesArray.length - 1}" class="message"><p>${msgString}</p><button>Delete</button></div>`;
```
In this way we're able to set the id of the message to a number using it's index in the array since new messages will always occupy the last index of the array, and we also inject the value of the msgString into a paragraph in the body of the message. I don't completely remember why I put a paragraph tag there, but it works on my end :+1:
And then similar to your line 15 I  used += to add outputString to the containerDiv.innerHTML